test/business-sync-shoebox-item-spec.js

```diff
@@ -178,7 +178,7 @@ describe('business-sync shoebox item document definition', function() {
         errorFormatter.typeConstraintViolation('annotations[metadata][0].modifications[0].source', 'object'),
         errorFormatter.datetimeFormatInvalid('annotations[metadata][0].modifications[0].timestamp'),
         errorFormatter.requiredValueViolation('annotations[metadata][0].modifications[1].source'),
-        errorFormatter.typeConstraintViolation('annotations[metadata][0].modifications[1].timestamp', 'datetime'),
+        errorFormatter.typeConstraintViolation('annotations[metadata][0].modifications[1].timestamp', 'string'),
```

Comment:
this file has a few cases where the `:` could be dropped to test the new custom timestamp validator